### DISCUSSION TITLE
pyMOR Community Meeting 2023-11-02

### DISCUSSION THREAD

**Discussion by pmli** [pymor/pymor#2208] [Announcements]
The next pyMOR Community Meeting will take place Thursday, November 2, at 13:00 CET and 8:00 EDT, online at:

Link: http://bbb.mpi-magdeburg.mpg.de/meeting/pymor
Password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- [x] Soft freeze (@sdrave)
- [x] bibtex formatting #2155 (@pmli)
- [x] QR PR #2211 and upcoming intern (@drittelhacker) 
- [x] Bilinear PR #2215 (@drittelhacker)
- [x] Progress bar #2216 (@HenKlei)
- [x] MORLAB algorithms #2213 (@pmli ?)

Moderator: @pmli

**Comment by lbalicki** [comment]
Notes:

- QR PR
  - How should QR and rank-revealing QR be handled?
  - Separate methods QR and RRQR or QR with an economy option
  - Having two separate methods seems to be preferred
  - State of QR interface PR seems good
  - Cholesky PR is ready for review
- Soft freeze
  - went through PRs
- Progress bar
  - PR currently using rich
  - will discuss offline how to possibly use logger
- MORLAB algorithms
  - will replace some of the pyMESS algorithms

Next Meeting: Tuesday 14/11/2023 at 15:00 CET, 9:00 EDT
